easybuild/tools/build_log.py

```diff
@@ -107,8 +107,9 @@ def nosupport(self, msg, ver):
 
     def error(self, msg, *args, **kwargs):
         """Print error message and raise an EasyBuildError."""
-        newMsg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg)
-        fancylogger.FancyLogger.error(self, newMsg, *args, **kwargs)
+        newMsg = ("EasyBuild crashed with an error %s: %s"
+                  % (self.caller_info(), (msg % args)))
```

Comment:
you only needed the interpolation for the `raise`, `fancylogger.FancyLogger.error` can handle `args` (but this is not a blocker for me)